### PR TITLE
Ding! Service bell was implemented

### LINKED ISSUE #5435: Ding! Add the Service Bell
- [x] The object of this issue is to add the service bell world object to the game, for mappers to place. The service bell sits on a tabletop at the kitchen or bar, and when clicked on, will emit a single, clear, ding.

- [x] Additionally, there is the big service bell, which should have a 0.5% chance of replacing any normal service bell at round start. Instead of the standard ding, it will emit a much larger bell dong! to reflect its bigger size and acoustic capability.

Sprites by Erinexx:
Service Bell: https://cdn.discordapp.com/attachments/409110435900424196/771411832211636315/unknown.png
BIG BELL: https://cdn.discordapp.com/attachments/402986688114065409/731227871321915603/unknown.png

Sounds:
Service Bell: https://puu.sh/GI5HD/6b05344dfc.ogg
BIG BELL: https://puu.sh/GI5Ic/7adc11e7f1.ogg

### PR DESCRIPTION
**Pull request by Lizurt**
### Purpose

closes #5435 

- Adds service bell to the game. BIG service bell wasn't implemented(I think this is superfluous, but it can be implemented later).
- - Service bell was added to all main maps(excluding TestStation) to the kitchen.
- - Service bell can be picked up via grab intent.

### Notes:

Sprites and sounds were provided in the attached issue

I hope I added everything I needed and didn't add anything extra(damn .mat files)

CL: Added service bell

### CODE REVIEW

**Comment by corp-0** [comment]
Fallstation is conflicting. Try getting the latest version of the repo and redo your changes to fall

**Comment by corp-0** [comment]
next time, if you want to avoid the headache that is conflicting scenes, just let the mappers add what you implemented

File: UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 9d1de2691495b304e9843ac3335b626d, type: 3}
+  m_Name: service_bell
+  m_EditorClassIdentifier: 
+  Variance:
+  - Frames:
+    - sprite: {fileID: 21300000, guid: 897babe988b324547a82d30795355c76, type: 3}
+      secondDelay: 0
+  IsPalette: 0
+  setID: 0
```
Comment (Lizurt) [inline UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: It looks like the sprite ID wasn't set automatically after all. I left it empty on the advice of Bod9001.

How can I fix this?

Comment (corp-0) [reply UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: leave empty and reimport the asset, or so I understood from what Bod said

Comment (Lizurt) [reply UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: If I leave it empty, the value becomes `0`. And it seems that reimport doesn't do anything useful to solve the problem.

Or maybe I'm doing something wrong?

1. Openning `Service_ball (Sprite Data SO)`.
2. Setting the Set ID value to blank(which becomes zero).
3. Right-clicking on the `service_ball` selected from the bottom.
4. Clicking reimport.

Comment (corp-0) [reply UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: then try deleting the sprite, meta and so. Do the process of importing the image into the project again.

Comment (Lizurt) [reply UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: Looks like it's fixed. Thanks

Comment (corp-0) [reply UnityProject/Assets/Textures/objects/machines/service_bell/service_bell.asset]: wonderful. Waiting for tests then I can merge. Thanks for the contribution!

**Pull request merged**